addons/grenades/functions/fnc_throwGrenade.sqf

```diff
@@ -37,9 +37,15 @@ if (local _unit) then {
     };
 
     if (getNumber (_config >> QGVAR(flashbang)) == 1) then {
-        private _fuzeTime = getNumber (_config >> "explosionTime");
-
-        [FUNC(flashbangThrownFuze), [_projectile], _fuzeTime] call CBA_fnc_waitAndExecute;
+        private _nBangs = [_config >> "nBangs", "number", 1] call CBA_fnc_getConfigEntry;
```

Comment:
```suggestion
private _nBangs = 1;

if (isNumber (_config >> QGVAR(nBangs)) then {
    _nBangs = getNumber (_config >> QGVAR(nBangs);
};
```
The config entry name written out would be `ace_grenades_nBangs`.

Reply:
Why not leave the line as it was but wrap nBangs in QGVAR? Does it throw an error if it exists but is the wrong type?

Reply:
There is no error. 

I just don't like `CBA_fnc_getConfigEntry`.